doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1382,32 +1354,34 @@ Managing the local context
            Undo.
            move H0 before H.
 
-.. tacn:: rename @ident__1 into @ident__2
-   :name: rename
+.. tacn:: rename {+, @ident__1 into @ident__2 }
 
-   This renames hypothesis :n:`@ident__1` into :n:`@ident__2` in the current
-   context. The name of the hypothesis in the proof-term, however, is left
```

Comment:
You think this note is not worth keeping?

Reply:
Missed it.  But I don't see why it isn't renamed in the proof term, too.  That seems like poor usability, just one more thing that may unnecessarily confuse users.  Worth filing an issue?

Reply:
I mean since it is in the proof term, users will not see it. Only if they inspect `Show Proof` will they seen that it has in fact not changed.

Reply:
Users at times do look at the proof term.  Unless it's technically too difficult, it should be consistent.

Reply:
> Worth filing an issue?

I guess you could open an issue or a Zulip topic to at least ask whether it's worth changing.